draft-ietf-rats-corim.md

```diff
@@ -825,6 +825,21 @@ $version-scheme /= int / text
 
 [^issue] https://github.com/ietf-rats-wg/draft-ietf-rats-corim/issues/8
```

Comment:
this should go now that we have text.

Reply:
fixed in branch - this thread is outdated.